src/shared/components/unsupportedBrowserModal/UnsupportedBrowserModal.tsx

```diff
@@ -14,24 +14,21 @@ export default class UnsupportedBrowserModal extends React.Component<{}, IBrowse
     constructor() {
         super();
 
-        let state;
         if (!browser)  {
```

Comment:
@alisman I think I just had this there as a fail safe - if for some reason the package did not work.  We can remove it though